gradle-plugin/settings.gradle.kts

```diff
@@ -21,3 +21,4 @@ dependencyResolutionManagement {
 }
 
 include(":plugin")
+rootProject.name = "emerge-gradle-plugin"
```

Comment:
this adds a rootproject name to remove a warning message in the console. (every project needs a rootProject.name)